spec/database_cleaner.rb

```diff
@@ -0,0 +1,6 @@
+require 'database_cleaner'
```

Comment:
you can create a `spec/spec_helper.rb` file and then you require this file on your specs files.

note that you will need #36 to clean the `test` database instead of the `dev` one.